spec/acceptance/06_postgresql_spec.rb

```diff
@@ -16,8 +16,7 @@ class { 'postgresql::globals':
       }
       class { 'postgresql::server':}
       class { 'sensu::backend':
-        license_source => '/root/sensu_license.json',
-        datastore      => 'postgresql',
```

Comment:
why is license_source no longer needed?

Reply:
Because all features are available to Sensu Go without a license until you reach 100 entities. It is something that changed in a more recent version of Sensu Go.  The license is only required if you have more than 100 entities. So I removed it from these tests since these tests are testing things like Postgresql and not the license which isn't necessary.